src/components/ClubCard/ClubCard.styles.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const ContentContainer = styled(Link)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background: var(--white);
+  border: 0.1rem solid var(--lightblue);
+  border-radius: var(--smallest-space);
+
+  max-width: 32.8rem;
+
+  transition: all 0.2s ease-in-out;
+  cursor: pointer;
+  &:hover,
+  &:active {
+    transform: translate(0rem, -0.7rem);
+    box-shadow: var(--card-shadow);
+  }
+  &:focus {
+    box-shadow: ${({ color }) => `0 0 0 0.3rem var(--${color})`};
+  }
+`;
+
+export const Image = styled.img`
```

Comment:
I think we have to add padding here as Figma has a bit of padding.

Reply:
I added 1rem padding to the container. But i must stress out that design seen on Figmas  _Prototypes and _Molecules pages are diferent. Its a confusing.